src/force/CabanaPD_ForceModels_LPS.hpp

```diff
@@ -0,0 +1,146 @@
+/****************************************************************************
+ * Copyright (c) 2022-2023 by Oak Ridge National Laboratory                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of CabanaPD. CabanaPD is distributed under a           *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef FORCE_MODELS_LPS_H
+#define FORCE_MODELS_LPS_H
+
+#include <CabanaPD_ForceModels.hpp>
+#include <CabanaPD_Types.hpp>
+
+namespace CabanaPD
+{
+template <>
+struct ForceModel<LPS, Elastic> : public BaseForceModel
+{
+    using base_type = BaseForceModel;
+    using base_model = LPS;
+    using fracture_type = Elastic;
+
+    using BaseForceModel::delta;
+
+    int influence_type;
+
+    double K;
+    double G;
+    double theta_coeff;
+    double s_coeff;
+
+    ForceModel(){};
+    ForceModel( const double _delta, const double _K, const double _G,
+                const int _influence = 0 )
```

Comment:
@streeve Why do we have "_influence = 0"?

Reply:
Default value